core/meta/src/TIsAProxy.cxx

```diff
@@ -39,19 +39,22 @@ namespace {
    {
       return (ClassMap_t::value_type*)p;
    }
+
 }
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Standard initializing constructor
 
 TIsAProxy::TIsAProxy(const std::type_info& typ)
-   : fType(&typ), fClass(nullptr), fLast(nullptr),
+   : fType(&typ), fClass(nullptr),
      fSubTypesReaders(0), fSubTypesWriteLockTaken(kFALSE),
-     fVirtual(kFALSE), fInit(kFALSE)
+     fInit(kFALSE), fVirtual(kFALSE)
 {
    static_assert(sizeof(ClassMap_t)<=sizeof(fSubTypes), "ClassMap size is to large for array");
 
    ::new(fSubTypes) ClassMap_t();
+   for(UInt_t i = 0; i < fgMaxLastSlot; ++i)
+      fLasts[i] = nullptr;
```

Comment:
> Also, why change the order of fInit and fVirtual (maybe to match declaration in the class)?

yes, the order was tweaked.

> Is it not possible to use `fLasts({nullptr, }), `to initialize all to nullptr and keep it above? 

Is that the actual syntax?  I.e. Do we need to write `nullptr` once or 8 times?

Reply:
Just once, that's C syntax, but I think it works fine in C++ as well to initialize arrays with a constant.

Reply:
`fLasts{}` is C++.

Reply:
Unforunately (probably because [until C++20 std::atomic does not initialize the data at all](https://en.cppreference.com/w/cpp/atomic/atomic/atomic)) ), neither options works.  `fLasts{}` leaves all values randoms and `fLasts{nullptr, }` only initialize the first value. ... well at least on my compiler (gcc 9.3).